### PR TITLE
Corrects RedHat Summit 2017 sample.

### LINKED ISSUE #231: Correct RedHatSummit17 sample.
Currently RedHatSummit17 sample is not able able to generate webapp using webapp generator.

### PR DESCRIPTION
**Pull request by agbilotia1998**
Fixes #231 #236

### CODE REVIEW

File: sample/RedHatSummit17/sponsors [hunk 1]
```diff
@@ -1,53 +1,53 @@
 [{
-		"logo":"/images/sponsors/dell.png",
+		"logo-url":"/images/sponsors/dell.png",
 		"description": "Dell EMC serves a key role in providing the essential infrastructure for organizations to build their digital future, transform IT and protect their most important asset, information. Dell EMC enables our enterprise customers\u2019 IT and digital business transformation through trusted hybrid cloud and big-data solutions, built upon a modern data center infrastructure that incorporates industry-leading converged infrastructure, servers, storage, and cybersecurity technologies.",
 		"level": "7",
 		"url": "http://www.dell.com/redhat",
 		"sponsor_type": "Platinum",
```
Comment (geekyd) [inline sample/RedHatSummit17/sponsors]: Please make it `type` refer to https://open-event-api.herokuapp.com/#sponsors. Make this change for all the sponsors.

File: sample/RedHatSummit17/speakers [hunk 2]
```diff
@@ -1,5 +1,5 @@
 [{
```
Comment (geekyd) [inline sample/RedHatSummit17/speakers]: Please refer to https://open-event-api.herokuapp.com/#speakers-speaker-get
Make changes to all the attributes.

File: sample/RedHatSummit17/sessions [hunk 3]
```diff
@@ -1,5 +1,5 @@
 [{
```
Comment (geekyd) [inline sample/RedHatSummit17/sessions]: Please compare the object to https://open-event-api.herokuapp.com/#sessions-sessions-details-get & make changes

**Comment by agbilotia1998** [comment]
@geekyd please check now.

File: sample/RedHatSummit17/speakers [hunk 4]
```diff
@@ -1,71 +1,71 @@
...
-      "photo": "/images/speakers/ma.jpg",
+      "photo-url": "/images/speakers/ma.jpg",
       "twitter": "https://twitter.com/michelleaalbers",
       "linkedin": "https://www.linkedin.com/in/michelle-aalbers-rtt-257b774a",
       "featured": "",
```
Comment (geekyd) [inline sample/RedHatSummit17/speakers]: Make it `is-featured` & add missing `long-biography `

File: sample/RedHatSummit17/speakers [hunk 5]
```diff
@@ -1,71 +1,71 @@
...
       "id": 1,
       "city": "",
       "organisation": "Schiphol Group",
       "heard_from": "",
```
Comment (geekyd) [inline sample/RedHatSummit17/speakers]: Make it `heard-from `

File: sample/RedHatSummit17/sessions [hunk 6]
```diff
@@ -9,27 +9,27 @@
...
-      "start_time": "2017-05-02T18:00:00-07:00",
-      "session_type": {
+      "starts-at": "2017-05-02T18:00:00-07:00",
+      "session-type": {
          "length": "03.00",
          "id": 3,
          "name": "3 hours session"
       },
       "comments": "",
       "slides": "",
```
Comment (geekyd) [inline sample/RedHatSummit17/sessions]: `slides-url`

**Comment by geekyd** [comment]
@agbilotia1998 Please compare all the objects to the API and make sure all the attributes are named as in the API

**Comment by agbilotia1998** [comment]
@geekyd please review.

File: sample/RedHatSummit17/event [hunk 7]
```diff
@@ -34,36 +34,36 @@
...
 		"logo": "",
-		"licence_url": "https://en.wikipedia.org/wiki/All_rights_reserved",
+		"licence-url": "https://en.wikipedia.org/wiki/All_rights_reserved",
 		"holder": "Facebook",
 		"licence": "All rights reserved",
-		"holder_url": null,
+		"holder-url": null,
 		"year": 2017
 	},
 	"call_for_papers": null,
```
Comment (geekyd) [inline sample/RedHatSummit17/event]: Use the new attribute here

File: sample/RedHatSummit17/sessions [hunk 8]
```diff
@@ -171,51 +171,51 @@
...
+         "heard-from": "",
          "small": "",
          "name": "Gonri Le Bouder",
          "thumbnail": "",
          "icon": ""
       }, {
          "city": "",
-         "short_biography": "",
+         "short-biography": "",
          "speaking_experience": "",
```
Comment (geekyd) [inline sample/RedHatSummit17/sessions]: Please use the new attribute here.

**Comment by agbilotia1998** [comment]
@geekyd can you please mention updated version of 'call_for_papers' attribute? Rest of all done.

**Comment by geekyd** [comment]
@agbilotia1998 It is `speakers-call` check https://open-event-api.herokuapp.com/#events

File: .idea/modules.xml [hunk 9]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (geekyd) [inline .idea/modules.xml]: Please remove this file

File: .idea/open-event.iml [hunk 10]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (geekyd) [inline .idea/open-event.iml]: Please remove this file

File: .idea/vcs.xml [hunk 11]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (geekyd) [inline .idea/vcs.xml]: Please remove this file

File: .idea/workspace.xml [hunk 12]
```diff
@@ -0,0 +1,472 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (geekyd) [inline .idea/workspace.xml]: Please remove this file

**Comment by agbilotia1998** [comment]
@geekyd done please review.

**Pull request merged**